### PR TITLE
feat(vehicles/roofs): updates vehicles roofs natives

### PR DESCRIPTION
**Pull request by ahcenezdh**
This PR follows #930 and completes the documentation for vehicles roof-related native functions.

* Update DOES_VEHICLE_HAVE_ROOF
* Update IS_VEHICLE_A_CONVERTIBLE
* Update LOWER_CONVERTIBLE_ROOF
* Update RAISE_CONVERTIBLE_ROOF
* Update SET_CONVERTIBLE_ROOF_LATCH_STATE
* Update SET_CONVERTIBLE_ROOF

### CODE REVIEW

**Comment by ahcenezdh** [comment]
@AvarianKnight pinging you so you can see this PR

**Comment by ahcenezdh** [comment]
ll examples are good, but the C# ones weren't tested

File: VEHICLE/DoesVehicleHaveRoof.md [hunk 1]
```diff
@@ -8,8 +8,44 @@ ns: VEHICLE
...
+    console.log("This vehicle has a roof.");
+} else {
+    console.log("This vehicle does not have a roof.");
+}
+```
+
+```cs
+using static CitizenFX.Core.Native.API;
+
+Vehicle vehicle = GetVehiclePedIsIn(PlayerPedId(), false);
```
Comment (AvarianKnight) [inline VEHICLE/DoesVehicleHaveRoof.md:45]:
```suggestion
int vehicle = GetVehiclePedIsIn(PlayerPedId(), false);
```

File: VEHICLE/IsVehicleAConvertible.md [hunk 2]
```diff
@@ -5,16 +5,48 @@ ns: VEHICLE
...
+    console.log("The vehicle is a convertible with an animatable roof.");
+} else {
+    console.log("The vehicle is not a convertible or does not have an animatable roof.");
+}
+```
+
+```cs
+using static CitizenFX.Core.Native.API;
+
+Vehicle vehicle = GetVehiclePedIsIn(PlayerPedId(), false);
```
Comment (AvarianKnight) [inline VEHICLE/IsVehicleAConvertible.md:46]:
```suggestion
int vehicle = GetVehiclePedIsIn(PlayerPedId(), false);
```

File: VEHICLE/LowerConvertibleRoof.md [hunk 3]
```diff
@@ -8,8 +8,35 @@ ns: VEHICLE
...
+```js
+const vehicle = GetVehiclePedIsIn(PlayerPedId(), false);
+if (!IsVehicleAConvertible(vehicle, false)) return;
+LowerConvertibleRoof(vehicle, false);
+```
 
+```cs
+using static CitizenFX.Core.Native.API;
+
+Vehicle vehicle = GetVehiclePedIsIn(PlayerPedId(), false);
```
Comment (AvarianKnight) [inline VEHICLE/LowerConvertibleRoof.md:39]:
```suggestion
int vehicle = GetVehiclePedIsIn(PlayerPedId(), false);
```

File: VEHICLE/RaiseConvertibleRoof.md [hunk 4]
```diff
@@ -8,8 +8,37 @@ ns: VEHICLE
...
+```js
+const vehicle = GetVehiclePedIsIn(PlayerPedId(), false);
+if (!IsVehicleAConvertible(vehicle, false)) return;
+RaiseConvertibleRoof(vehicle, false);
+```
+
+```cs
+using static CitizenFX.Core.Native.API;
+
+Vehicle vehicle = GetVehiclePedIsIn(PlayerPedId(), false);
```
Comment (AvarianKnight) [inline VEHICLE/RaiseConvertibleRoof.md:41]:
```suggestion
int vehicle = GetVehiclePedIsIn(PlayerPedId(), false);
```

File: VEHICLE/SetConvertibleRoof.md [hunk 5]
```diff
@@ -29,18 +36,21 @@ This allows for the vehicle's roof to be put on when set to true, and removed wh
...
 // In this case we are removing the roof from the vehicle.
 const vehicle = GetVehiclePedIsIn(PlayerPedId(), false);
+if (!DoesVehicleHaveRoof(vehicle)) return;
 SetConvertibleRoof(vehicle, false);
 ```
 
 ```cs
 // In this case we are removing the roof from the vehicle.
 using static CitizenFX.Core.Native.API;
 Vehicle vehicle = GetVehiclePedIsIn(PlayerPedId(), false);
```
Comment (AvarianKnight) [inline VEHICLE/SetConvertibleRoof.md:53]:
```suggestion
int vehicle = GetVehiclePedIsIn(PlayerPedId(), false);
```

File: VEHICLE/SetConvertibleRoofLatchState.md [hunk 6]
```diff
@@ -6,11 +6,39 @@ aliases: ["0x1A78AD3D8240536F"]
...
+const vehicle = GetVehiclePedIsIn(PlayerPedId(), false);
+if (!DoesVehicleHaveRoof(vehicle)) return;
+SetConvertibleRoofLatchState(vehicle, true);
+```
+
+```cs
+// To latch (lock) the convertible roof, preventing it from being opened
+using static CitizenFX.Core.Native.API;
 
+Vehicle vehicle = GetVehiclePedIsIn(PlayerPedId(), false);
```
Comment (AvarianKnight) [inline VEHICLE/SetConvertibleRoofLatchState.md:41]:
```suggestion
int vehicle = GetVehiclePedIsIn(PlayerPedId(), false);
```

**Review by AvarianKnight — COMMENTED** [review]
CS examples are broken

**Pull request merged**